dev/VSIX/Extension/ExtensionPack.vsext

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Read about creating extension packs at https://aka.ms/vsextpack",
```

Comment:
should update this from the default, to something like "Project Reunion experimental extensions"